localstack/services/lambda_/invocation/logs.py

```diff
@@ -62,10 +63,15 @@ def run_log_loop(self, *args, **kwargs) -> None:
                 )
 
     def start_subscriber(self) -> None:
+        if not is_api_enabled("logs"):
```

Comment:
just preventing the thread start here. we can discuss if there are better approaches to handle this when `logs` are disabled.